integrations/whatsapp/src/util.ts

```diff
@@ -20,3 +20,11 @@ export function chunkArray<T>(array: T[], chunkSize: number) {
 export async function sleep(ms: number) {
   return new Promise((resolve) => setTimeout(resolve, ms))
 }
+
+export function truncate(input: string, maxLength: number) {
```

Comment:
I added this function as a minor UX improvement so that text that gets truncated due to Whatsapp's limits shows a ellipsis to avoid it just appearing cut off.